src/rest_framework_reactive/__init__.py

```diff
@@ -1,3 +1,3 @@
 default_app_config = (
-    'rest_framework_reactive.apps.BaseConfig'
-)  # pylint: disable=invalid-name
+    'rest_framework_reactive.apps.BaseConfig'  # pylint: disable=invalid-name
```

Comment:
This package actually no longer uses Pylint, so you can just remove this comment - and also [all other occurrences](https://github.com/genialis/django-rest-framework-reactive/search?q=pylint&unscoped_q=pylint).